demosplan/DemosPlanCoreBundle/ResourceTypes/OrgaResourceType.php

```diff
@@ -123,7 +128,13 @@ public function getAccessCondition(): PathsBasedInterface
             );
         }
 
-        $organisationId = $this->currentUser->getUser()->getOrga()->getId();
+        $user = $this->currentUser->getUser();
+        if (null === $user->getOrga()) {
+            $this->logger->error('User without organisation.', [$user->getId()]);
+            throw new MissingOrganisationException();
```

Comment:
I don't think an exception is necessary. If the user has no organisation (for whatever reason) he can be simply denied access to any `Orga` resource.

Reply:
b369b8a75d9506561f971e993598b86f2fcc35ed

Reply:
I don't see the connection between your changes in https://github.com/demos-europe/demosplan-core/commit/955b86ab23be1f9906ee39e59a5d7bb2da5ce38e and my concern.

Reply:
solved by https://github.com/demos-europe/demosplan-core/commit/cfede6ed04ccf06c71898443b2c74c68bd4e2d74